frontend/components/ResponsiveImage.tsx

```diff
@@ -1,46 +1,13 @@
-import React, { useState, useCallback, useEffect } from 'react';
 import { Box } from '@mui/system';
-import { Grid } from '@mui/material';
-
-const useMediaQuery = (width: number) => {
-  const [targetReached, setTargetReached] = useState(false);
-
-  const updateTarget = useCallback((e: MediaQueryListEvent) => {
-    if (e.matches) {
-      setTargetReached(true);
-    } else {
-      setTargetReached(false);
-    }
-  }, []);
-
-  useEffect(() => {
-    const media = window.matchMedia(`(max-width: ${width}px)`);
-    media.addEventListener('change', updateTarget);
-
-    // Check on mount (callback is not called until a change occurs)
-    if (media.matches) {
-      setTargetReached(true);
-    }
-
-    return () => media.removeEventListener('change', updateTarget);
-  }, []);
-
-  return targetReached;
-};
+import useMediaQuery from '../utils/responsive';
```

Comment:
Is there a reason why material UI's [`useMediaQuery`](https://mui.com/material-ui/react-use-media-query/) doesn't work?

Reply:
In fact there's no reason to use media query at all:

[Hiding boxes on breakpoints](https://mui.com/system/display/#hiding-elements)

Just use sm instead of the 768,

Reply:
both works. I'll use useMediaQuery from mui